src/Core/App.php

```diff
@@ -74,6 +74,92 @@ public static function className($class, $type = '', $suffix = '')
     }
 
     /**
+     * Returns the plugin split name of a class
+     *
+     * Examples:
+     *
+     * ```
+     * App::pluginSplitName(
```

Comment:
I agree, `pluginSplit` usually returns an array of split elements, not a string.
Since the main method is `App::className()` I would go with `App::shortClassName()` or `App::splitClassName()` to be explicit about what it does.